src/test/java/com/epam/tc/hw6/step/ActionStep.java

```diff
@@ -0,0 +1,136 @@
+package com.epam.tc.hw6.step;
+
+import com.epam.tc.hw3.page.objects.composite.DifferentElementsPage;
+import com.epam.tc.hw3.page.objects.composite.HomePage;
+import com.epam.tc.hw3.page.objects.composite.component.FrameComponent;
```

Comment:
Why `hw3` package is used instead of `hw6`

Reply:
hw3 changed to hw6